src/GitTfs/Core/TfsChangeset.cs

```diff
@@ -178,12 +178,20 @@ private LogEntry MakeNewLogEntry(IChangeset changesetToLog, IGitTfsRemote remote
             {
             }
             var name = changesetToLog.Committer;
+            var key = changesetToLog.Committer.Split('\\');
             var email = changesetToLog.Committer;
+            Trace.WriteLine("changesetToLog.Committer=[" + changesetToLog.Committer + "]");
             if (_authors != null && _authors.Authors.ContainsKey(changesetToLog.Committer))
             {
                 name = _authors.Authors[changesetToLog.Committer].Name;
                 email = _authors.Authors[changesetToLog.Committer].Email;
             }
+            else if (_authors != null && _authors.Authors.ContainsKey(key.Last()) )
```

Comment:
Could you explain which case you solve here?

Reply:
This should not have been included in this PR.
It attempts to key off of usernames without specifying the AD sub-domain in the map file.

Reply:
removed